### ISSUE TITLE
⌨ Heading `MenuButton` on `RichTextInput` insufficient color contrast

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#15649]
### ⌨ Accessibility and Expected Behaviour

Heading `MenuButton` on `RichTextInput` insufficient color contrast.

### WCAG 2.1 Criteria

> Element has insufficient color contrast of 2.11 (foreground color: #553874, background color: #06070a, font size: 10.5pt (14px), font weight: bold). Expected contrast ratio of 4.5:1.

### 📸 Screenshot

<img width="1878" height="898" alt="Image" src="https://github.com/user-attachments/assets/3c3e4e1d-e239-4345-b140-78be3f5136fc" />
<img width="1460" height="668" alt="Image" src="https://github.com/user-attachments/assets/5aa220a7-b040-43cf-94ec-9d7af4e48877" />
<img width="1462" height="714" alt="Image" src="https://github.com/user-attachments/assets/a7a60716-dcb9-420f-b34a-88dbdd7c7115" />

### 🙋‍♀️ Proposed Solution

We should match our styling for "active" elsewhere on the site (tabs, menu, etc...)

Do do so:
1. Remove all the bolding from the buttons
2. Remove the "active" color from the active button
3. Bold the active button and remove underline from it

In addition to this, if this is deemed insufficient, we can use the active colors from the main nav buttons.

### ✅ Acceptance Criteria

- [ ] `MenuButton` style for heading works the same as others

**Comment by gobyrne** [comment]
From Josh:
We should match our styling for "active" elsewhere on the site (tabs, menu, etc...)

Do do so:
1. Remove all the bolding from the buttons
2. Remove the "active" color from the active button
3. Bold the active button and remove underline from it

In addition to this, if this is deemed insufficient, we can use the active colors from the main nav buttons.

I added this as a proposed solution.
